### PR TITLE
Adding Binance integration.

### PR DESCRIPTION
**Pull request by Wingman4l7**
### Description
Adding Binance integration.

##### Checklist

- [x] `yarn test:ci` passes
- [x] tests are included

### CODE REVIEW

File: src/services/dragonchain-client/DragonchainClient.ts [hunk 1]
```diff
@@ -1076,6 +1077,127 @@ export class DragonchainClient {
...
+    /**
+     * The name of the network to update
+     */
+    name: string;
+    /**
+     * Whether or not this is a testnet wallet/address.  Defaults to True.
+     */
+    testnet?: boolean;
+    /**
+     * The base64 or hex encoded private key to use. Will automatically generate a random one if not provided
```
Comment (cheeseandcereal) [inline src/services/dragonchain-client/DragonchainClient.ts]: can also take mnemonic

File: src/services/dragonchain-client/DragonchainClient.ts [hunk 2]
```diff
@@ -1076,6 +1077,127 @@ export class DragonchainClient {
...
+     */
+    name: string;
+    /**
+     * the amount of token to send with this transaction
+     */
+    amount: number;
+    /**
+     * The (hex-encoded) address to send the transaction to
+     */
+    to_address: string;
```
Comment (cheeseandcereal) [inline src/services/dragonchain-client/DragonchainClient.ts]: this should be camel case

File: src/services/dragonchain-client/DragonchainClient.ts [hunk 3]
```diff
@@ -748,8 +749,8 @@ export class DragonchainClient {
       if (!process.env.SMART_CONTRACT_ID) throw new FailureByDesign('PARAM_ERROR', 'Parameter `smartContractId` is required when not running within a smart contract');
       options.smartContractId = process.env.SMART_CONTRACT_ID;
     }
-    const response = (await this.get(`/v1/get/${options.smartContractId}/${options.key}`, false)) as unknown;
-    return response as string;
+    const response = (await this.get(`/v1/get/${options.smartContractId}/${options.key}`, false)) as Response<string>;
```
Comment (cheeseandcereal) [inline src/services/dragonchain-client/DragonchainClient.ts]:
```suggestion
    return (await this.get(`/v1/get/${options.smartContractId}/${options.key}`, false)) as Response<string>;
```

File: src/services/dragonchain-client/DragonchainClient.ts [hunk 4]
```diff
@@ -748,8 +749,8 @@ export class DragonchainClient {
...
+    return response;
```
Comment (cheeseandcereal) [inline src/services/dragonchain-client/DragonchainClient.ts]:
```suggestion
```

File: src/services/dragonchain-client/DragonchainClient.ts (hunk 1 above)
Comment (Wingman4l7) [reply src/services/dragonchain-client/DragonchainClient.ts]: Fixed.

File: src/services/dragonchain-client/DragonchainClient.ts (hunk 3 above)
Comment (Wingman4l7) [reply src/services/dragonchain-client/DragonchainClient.ts]: Fixed.

File: src/services/dragonchain-client/DragonchainClient.ts (hunk 4 above)
Comment (Wingman4l7) [reply src/services/dragonchain-client/DragonchainClient.ts]: Fixed.

File: src/services/dragonchain-client/DragonchainClient.ts (hunk 2 above)
Comment (Wingman4l7) [reply src/services/dragonchain-client/DragonchainClient.ts]: Fixed.

**Pull request merged**